org-owners-guide.md

```diff
@@ -37,8 +37,18 @@ project:
 * [kubernetes-graveyard](https://github.com/kubernetes-graveyard) †
 * [kubernetes-helm](https://github.com/kubernetes-helm)
 * [kubernetes-incubator](https://github.com/kubernetes-incubator)
+* [kubernetes-incubator-retired](https://github.com/kubernetes-incubator-retired)
 * [kubernetes-retired](https://github.com/kubernetes-retired)
 * [kubernetes-security](https://github.com/kubernetes-security)
+* [kubernetes-sig-testing](https://github.com/kubernetes-sig-testing)
+* [kubernetes-providers](https://github.com/kubernetes-providers)
+* [kubernetes-addons](https://github.com/kubernetes-addons)
```

Comment:
I would prefer we differentiate between orgs we are actively using, vs orgs which we are holding for potential future use

Reply:
In the original list I was trying to only list the orgs we are using and not the ones we are name squatting on. For example, we aren't using kubernetes-charts. You'll noticed the section title is "Current Organizations In Use". If there are no repos on an org or no immediate plans to start adding them can we not list them?

Reply:
@spiffxp @mattfarina SGTM.

I'll split the list into two categories, but have to reconfirm:

### Actively used GitHub Organizations:

### Non-actively used GitHub Organizations:

Reply:
second list has kuberentes-providers twice and is non alphabetized, but otherwise LGTM